platform/core/src/utils/guid.crypto.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * Create a random GUID using Web Crypto API for better randomness
+ *
+ * @return {string}
+ */
+const guid = () => {
```

Comment:
why we are adding yet another uid to our platform? we already have guid and uuiv4.

Reply:
If you want crypto the uuiv4 already is crypto based

Reply:
Sounds good, I'll be updating it to use it.